doc/getting_started/setup_fpga.md

```diff
@@ -45,6 +45,16 @@ This can be spliced into the bitstream to overwrite the testing boot ROM as desc
 However, if you do not want to do the splicing yourself, both versions of the bitstream are available in the download as `*.bit.orig` and `*.bit.splice` (containing the test ROM and the ROM respectively).
 The metadata for the latest bitstream (the approximate creation time and the associated commit hash) is also available as a text file and can be [downloaded separately](https://storage.googleapis.com/opentitan-bitstreams/master/latest.txt).
 
+### Using the `@bitstreams` repository
+
+OpenTitan's build system automatically fetches pre-built bitstreams via the `@bitstreams` repository.
+
+To keep the `@bitstreams` repository in sync with the current Git revision, install the Git post-checkout hook:
+
+```sh
+cp util/git/hooks/post-checkout .git/hooks/
```

Comment:
nit: I know this depends on individual workflows and preferences but should we present symlink as another (maybe  the preferred) alternative? This hook is pretty small but symlinking (assuming that the local clone is at a stable location) would auto-update the hook if it ever changes.

Reply:
Interesting thought! I like being automatic, but we should think about how that interacts with git worktrees.

Reply:
Good point, I think you'd want to use the _actual_ clone not the worktree for this. Just something to keep in mind and shouldn't block this PR and everyone can choose the best option that suits their needs and workflows. I'd imagine we'd announce any changes in the hook either way.

Reply:
Thinking about this a little more, I wonder what happens when a git hook symlinks to a nonexistent file. That could be an issue if we use the symlink approach and want to check out older commits. Anyway, off-topic for this PR :)